library/src/tests/resources/src/qpe.qs

```diff
@@ -0,0 +1,49 @@
+namespace Test {
+    import Std.Diagnostics.*;
+
+    operation TestQPE_Phase(gate : Qubit => Unit is Ctl + Adj, power : Int) : Unit {
+        use state = Qubit();
+        use phase = Qubit[4];
+
+        // Estimate eigenvalue of gate on eigenvector of |0⟩
+        ApplyQPE(ApplyOperationPowerCA(_, qs => gate(qs[0]), _), [state], phase);
+        // It should be 1, which corresponds to the binary fraction 0.0000 of 2π.
```

Comment:
"Binary fraction X of Y" is not a phrasing I'm familiar with. Can you explain what that means?

Reply:
This comment is part of a particular test and is very much in the context of ApplyQPE operation. I didn't do a good job explaining it here thoroughly. I'll provide a better explanation.

Reply:
Updated.